pkg/model/elbv2/target_group.go

```diff
@@ -121,6 +121,10 @@ type TargetGroupSpec struct {
 	// The protocol to use for routing traffic to the targets.
 	Protocol Protocol `json:"protocol"`
 
+	// The target group protocol version.
+	// +optional
+	ProtocolVersion ProtocolVersion `json:"protocolVersion,omitempty"`
```

Comment:
let's make this to be `*ProtocolVersion`. all optional field should be *Type. even for types with sensible zero value.